### PR TITLE
Pim 7199 : Display a message when trying to delete the pim identifier attribute

### PR DESCRIPTION
**Pull request by christophe-chausseray**
**Description (for Contributor and Core Developer)**

Currently, when you try to delete an identifier argument you will have a 500.

So this fix permits to display a message ("The attribute identifier cannot be deleted") when we try to delete an identifier attribute.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | OK
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | OK
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: features/attribute/delete_attribute.feature [hunk 1]
```diff
@@ -38,3 +36,13 @@ Feature: Delete an attribute
...
+
+  @jira https://akeneo.atlassian.net/browse/PIM-7199
+  Scenario: An identifier attribute cannot be deleted
+    Given the "default" catalog configuration
+    And I am logged in as "Julia"
+    When I am on the attributes page
+    And I click on the "delete" action of the row which contains "SKU"
+    And I confirm the deletion
+    Then I should see the text "Identifier attribute can not be removed"
+    And there should be a "SKU" attribute
```
Comment (willy-ahva) [inline features/attribute/delete_attribute.feature]: add a blank line at the end of the file

Comment (damien-carcel) [inline features/attribute/delete_attribute.feature]: Missing EOF.

Comment (christophe-chausseray) [reply features/attribute/delete_attribute.feature]: done

Comment (christophe-chausseray) [reply features/attribute/delete_attribute.feature]: done

**Pull request merged**